src/pages/admin-ui-sdk/configuration.md

```diff
@@ -48,7 +48,13 @@ bin/magento cache:clean admin_ui_sdk
 
 ## Test with a sample app
 
-You can download a sample app from the [Commerce UI test extension repository](https://github.com/adobe/adobe-commerce-samples/tree/main/sample-extension) to gain insight on how the Admin SDK injects menus and pages into the Admin.
+### Prerequisites
+
+* Adobe Commerce instance installed on local machine.
+
+### Configuration
+
+You can download a sample app from the [Adobe Commerce Samples repository](https://github.com/adobe/adobe-commerce-samples/tree/main/admin-ui-sdk/menu/custom-menu) to gain insight on how the Admin SDK injects menus and pages into the Admin.
```

Comment:
The link returns `404 - page not found`

Reply:
Yes, this is expected as I want to merge both PRs (https://github.com/adobe/adobe-commerce-samples/pull/9) at the same time to avoid broken links.